### PR TITLE
Add pydantic classes

### PR DESCRIPTION
**Pull request by lokeshwarlakhi**
Added pydantic classes , the pydantic is working fine with:

- [x] fraud detection 
- [ ] human orientation classification
- [ ] ride time estimation
- [ ] shopping cart recommendation
- [ ] text summarization
- [ ] finetuning LLM
- [ ] conversation summarization

so far so good , working properly (work under progress)

### CODE REVIEW

**Review by Dominastorm — CHANGES_REQUESTED** [review]
Recommended some changes to improve code readability. There is one bug in the last cell of examples/fraud_detection/run.ipynb. Works properly after fixing it. Otherwise, looks good!

File: examples/fraud_detection/run.ipynb
[the remark this answers is not in the document]
Comment (vipgupta) [reply examples/fraud_detection/run.ipynb:1]: Makes sense!

File: uptrain/core/classes/managers/check_manager.py [hunk 1]
```diff
@@ -26,12 +26,12 @@ def __init__(self, framework, checks=[]):
         self.visuals_to_check = []
         self.fw = framework
         for check in checks:
-            if check["type"] in Monitor:
-                self.add_monitor(check)
-            if check["type"] in Statistic:
-                self.add_statistic(check)
-            if check["type"] in Visual:
-                self.add_visual(check)
+            if check.dict()["type"] in [monitor.value for monitor in Monitor]:
```
[the remark this answers is not in the document]
Comment (vipgupta) [reply uptrain/core/classes/managers/check_manager.py:29]: Agreed! Would `check.type` not work?

[the remark this answers is not in the document]
Comment (lokeshwarlakhi) [reply uptrain/core/classes/managers/check_manager.py:29]: Yes check.type is working fine.

File: examples/fraud_detection/run.ipynb
[the remark this answers is not in the document]
Comment (lokeshwarlakhi) [reply examples/fraud_detection/run.ipynb:1]: cfg is the right way in this, because that's how the config is initialized

**Pull request closed without merging**